sdk/src/main/java/ch/admin/bag/covidcertificate/sdk/android/models/VerifierCertificateHolder.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021 Ubique Innovation AG <https://www.ubique.ch>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package ch.admin.bag.covidcertificate.sdk.android.models
+
+import ch.admin.bag.covidcertificate.sdk.core.models.healthcert.CertificateHolder
+import ch.admin.bag.covidcertificate.sdk.core.models.healthcert.CovidCertificate
+
+class VerifierCertificateHolder(
+	internal val certificateHolder: CertificateHolder
+) : CovidCertificate by certificateHolder.certificate
```

Comment:
From the naming I would have expected `VerifierCertificateHolder` to inherit from `CertificateHolder` rather than from `CovidCertificate`.
It makes sense since we want to restrict the data exposed to the verifier, but maybe this is worth a comment - anybody reading this in the future will otherwise be confused too, it even took me a minute to understand the reasoning :)